src/main/java/org/jabref/gui/preferences/keybindings/presets/NewEntryBindingPreset.java

```diff
@@ -0,0 +1,40 @@
+package org.jabref.gui.preferences.keybindings.presets;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jabref.gui.keyboard.KeyBinding;
+
+public class NewEntryBindingPreset implements KeyBindingPreset {
+
+    private static final Map<KeyBinding, String> KEY_BINDINGS = new HashMap<>();
+
+    static {
+        // Clear conflicting default presets
+        KEY_BINDINGS.put(KeyBinding.PULL_CHANGES_FROM_SHARED_DATABASE, "");
+        KEY_BINDINGS.put(KeyBinding.COPY_PREVIEW, "");
+
+        // Add new entry presets
+        KEY_BINDINGS.put(KeyBinding.NEW_ARTICLE, "Ctrl+shift+A");
+        KEY_BINDINGS.put(KeyBinding.NEW_BOOK, "Ctrl+shift+B");
+        KEY_BINDINGS.put(KeyBinding.NEW_ENTRY, "Ctrl+N");
+        KEY_BINDINGS.put(KeyBinding.NEW_ENTRY_FROM_PLAIN_TEXT, "Ctrl+shift+N");
+        KEY_BINDINGS.put(KeyBinding.NEW_INBOOK, "Ctrl+shift+I");
+        KEY_BINDINGS.put(KeyBinding.NEW_INPROCEEDINGS, "Ctrl+shift+C");
+        KEY_BINDINGS.put(KeyBinding.NEW_MASTERSTHESIS, "Ctrl+shift+M");
+        KEY_BINDINGS.put(KeyBinding.NEW_PHDTHESIS, "Ctrl+shift+T");
+        KEY_BINDINGS.put(KeyBinding.NEW_PROCEEDINGS, "Ctrl+shift+P");
+        KEY_BINDINGS.put(KeyBinding.NEW_TECHREPORT, "Ctrl+shift+R");
+        KEY_BINDINGS.put(KeyBinding.NEW_UNPUBLISHED, "Ctrl+shift+U");
+    }
+
+    @Override
+    public String getName() {
+        return "New Entries";
```

Comment:
I find "new entries" a bit confusing, maybe something longer that better explains the purpose? Should be localized as well.

Reply:
Please make a suggestion. We did not come up with something better yet.

Reply:
New Entry types ?

Reply:
Something like "Bindings for creating all entry types"?

Reply:
Remember this is just shown in the small menubutton below the the keybindings table
![image](https://user-images.githubusercontent.com/50491877/117551748-f08ef780-b047-11eb-8070-65c61f376d14.png)

Reply:
A long text would just repeat information the user already has and would be imho extremely annoying.

Reply:
Yeah, I couldn't find a nice formulation that doesn't include "Bindings". But with "New entries" or "New entry types" it's really not clear what this is doing.